root/static/scripts/common/MB/Control/Autocomplete.js

```diff
@@ -464,8 +464,8 @@ $.widget("mb.entitylookup", $.ui.autocomplete, {
             action: _.bind(this._searchAgain, this, true)
         });
 
-        var allowCreation = window === window.top,
-            entity = this.entity.replace("-", "_");
+        var allowCreation = window === window.top;
+            var entity = this.entity.replace("-", "_");
```

Comment:
You should indent this line the same way as the previous var :) Same for all the other cases in this commit.